libraries/redcore/layouts/toolbar/button/versions.php

```diff
@@ -0,0 +1,28 @@
+<?php
+/**
+ * @package     Redcore
+ * @subpackage  Layout
+ *
+ * @copyright   Copyright (C) 2008 - 2015 redCOMPONENT.com. All rights reserved.
+ * @license     GNU General Public License version 2 or later, see LICENSE.
+ */
+
+defined('JPATH_REDCORE') or die;
+
+$height    = $displayData['height'];
+$width     = $displayData['width'];
+$itemId    = (int) $displayData['itemId'];
+$typeId    = (int) $displayData['typeId'];
+$typeAlias = $displayData['typeAlias'];
+$title     = $displayData['title'];
+
+$link = 'index.php?option=com_contenthistory&amp;view=history&amp;layout=modal&amp;tmpl=component&amp;item_id=' . $itemId .
```

Comment:
this is very specific, it can be used only for versions and nothing more...
why couldn't you just expand link or modal buttons for this?